ProtonVPNTests/ViewModels/Signup/SignUpFormViewModelTests.swift

```diff
@@ -111,7 +111,7 @@ class SignUpFormViewModelTests: XCTestCase {
         viewModel.password2 = "abc"
         
         userApiService.callbackcheckAvailability = { username, success, failure in
-            failure(ApiError.uknownError)
+            failure(ApiError.unknownError)
```

Comment:
This presumably will break as it appears to be dependent on some external thing. I'm not sure where it is...